app/main.py

```diff
@@ -21,7 +21,7 @@ async def lifespan(_: FastAPI) -> AsyncIterator[None]:
 
 
 app = FastAPI(lifespan=lifespan)
-
+pv_base_url = "https://phinvads.cdc.gov/baseStu3"
```

Comment:
niceeee

Reply:
when you merge in main, could you refactor the other endpoints to use this variable?